### PR TITLE
Add Verrazzano track to site

### PR DESCRIPTION
**Pull request by mlittlec**

### CODE REVIEW

File: content/vz/vz.md
Comment (bgraef) [inline content/vz/vz.md:1]: - double Oracle in banner alt text in line 26
- swap to Verrazzano in line 32, 44, 50, 56, 62, 68

**Review by bgraef — COMMENTED** [review]
made suggestions for vz.md file

File: content/vz/vz.md
Comment (mlittlec) [reply content/vz/vz.md:1]: Requested updates committed

Comment (bgraef) [reply content/vz/vz.md:1]: Updates look good 👍

**Review by bgraef — COMMENTED** [review]
looks good 👍

**Review by bgraef — APPROVED** [review]
🚀

**Pull request merged**